backend/app/models.py

```diff
@@ -253,7 +253,7 @@ class DocumentPublic(BaseModel):
     genesis: str | None = None
     access: DocumentShareStatus = DocumentShareStatus.edit
     color_scheme: list[str] | None = None
-    map_type: str | None = None
+    map_type: str = "default"
```

Comment:
this should really come from the db. Are there cases where the map type is unset and needs a default?

```suggestion
    map_type: str
```